packages/frontend-api/src/Resources/config/graphql-types/ModelType/Customer/CustomerUser/Input/RegistrationByOrderInputDecorator.types.yaml

```diff
@@ -0,0 +1,20 @@
+RegistrationByOrderInputDecorator:
+    type: input-object
+    decorator: true
+    config:
+        fields:
+            orderUrlHash:
```

Comment:
should we use urlhash or uuid? in `AddOrderItemsToCartInputDecorator` we use UUID... I'm not sure where is the line of using one or the other

Reply:
discussed, urlHash is a bit more secure